picker/Rofimoji.py

```diff
@@ -173,36 +171,26 @@ def resolve_all_files(self):
         file_names = self.args.files
 
         if len(file_names) == 1 and file_names[0] == 'all':
-            file_names = [os.path.splitext(file)[0] for file in
-                          os.listdir(os.path.join(os.path.dirname(__file__), "data"))
-                          if fnmatch.fnmatch(file, "*.csv")]
+            file_names = [file.stem for file in (Path(__file__).parent / "data").glob("*.csv")]
         return file_names
 
-    def load_from_file(self, file_name: str) -> str:
-        provided_file = os.path.join(os.path.dirname(__file__), "data", file_name + '.csv')
-        if os.path.isfile(file_name):
+    def load_from_file(self, file_name: Path) -> str:
```

Comment:
This can't be a `Path`. In the (likely much more common) case that this references a built-in list, this is just a file name without extension.

Reply:
You're right, I reviewed the logic of this method and adapted it to an expected `str` that _might_ be a file.

Reply:
I'm really sorry, but to me, `file_name` is still shown as a `Path`. Is Github wonky and not showing me some changes? Or did I miss something else?

Reply:
Sorry, probably wrong rebase (and maybe the master/main difference). Pushed again.